src/eterna/pose2D/Pose2D.ts

```diff
@@ -927,11 +1023,15 @@ export default class Pose2D extends ContainerObject implements Updatable {
             this._shiftStart = -1;
             this._shiftEnd = -1;
             this.updateShiftHighlight();
-        } else if (this.annotationManager.annotationModeActive) {
-            this._annotationRanges = [];
-            this._selectingAnnotationRange = false;
-            this.updateAnnotationRangeHighlight();
         }
+        // else if (this.annotationManager.annotationModeActive) {
+        //     // Clicked ether/cytoplasm space while in annotation mode
+        //     // Deselect any selected base ranges
+        //     this._annotationRanges = [];
+        //     this.hideAnnotationContextMenu();
+        //     this._selectingAnnotationRange = false;
+        //     this.updateAnnotationRangeHighlight();
+        // }
```

Comment:
```suggestion
```
I assume this isn't needed

Reply:
I'd kept it in case players preferred a slightly different mechanic when specifying bases. The current implementation won't clear your base selection if you click empty space, you need to press cancel to do that. The benefit of that is that it avoids slips that could erroneously clear one's selection, so I opted to select the approach that requires more explicit cancellation. It's probably fair to remove though. It's not too hard to reintroduce later.